jpegxl-rs/src/encode.rs

```diff
@@ -92,6 +92,17 @@ impl From<ColorEncoding> for JxlColorEncoding {
     }
 }
 
+/// Encoding box types
+#[derive(Debug, Clone, Copy)]
+pub enum BoxTypes {
```

Comment:
How about a `Metadata<'data>` with `Exif<&'data u8>` and etc. ?

Reply:
Got it! We can use `Metadata` to collect multiple metadata contents and other metadata configuration.

But maybe we can remain this to use in box type matching with `c_char` when adding box.